pandas/core/arrays/interval.py

```diff
@@ -1985,6 +1985,9 @@ def _from_combined(self, combined: np.ndarray) -> IntervalArray:
             )._from_sequence(nc[:, 1], dtype=dtype)
         else:
             assert isinstance(dtype, np.dtype)
+            nc = np.hstack(
```

Comment:
is the hstack really necessary here since the next 2 lines is just splitting them back up?